### PR TITLE
GDB-6401 Display cluster nodes

### PR DESCRIPTION
**Pull request by yordanalexandrov**
- added visualization of cluster nodes
 - styled nodes as per node status
 - added links between nodes
 - styled links as per link status

### CODE REVIEW

File: src/js/angular/clustermanagement/cluster-drawing.service.js [hunk 1]
```diff
@@ -67,3 +71,188 @@ export function moveElement(element, x, y) {
...
+                    hostname = parser.hostname.split('.')[0];
+                }
+                return hostname + ":" + parser.port;
+            }
+        });
+
+    nodes
+        .select('.id-host-background')
+        .attr('width', function (d) {
+            return d3.select(d.labelNode).node().getBBox().width + 10;
```
Comment (svilenvelikov) [inline src/js/angular/clustermanagement/cluster-drawing.service.js:155]: what are these magic numbers +10, +2, -5, -1? Can you drop a comment or better make some constants maybe

File: src/js/angular/clustermanagement/cluster-drawing.service.js [hunk 2]
```diff
@@ -67,3 +71,188 @@ export function moveElement(element, x, y) {
...
+            return d3.select(d.labelNode).node().getBBox().x - 5;
+        })
+        .attr('y', function (d) {
+            return d3.select(d.labelNode).node().getBBox().y - 1;
+        })
+        .attr('fill', '#EEEEEE');
+}
+
+function getNodeIconType(node) {
+    if (node.nodeState === 'LEADER') {
```
Comment (svilenvelikov) [inline src/js/angular/clustermanagement/cluster-drawing.service.js:170]: you can create enum like objects with constants for these states

Comment (yordanalexandrov) [reply src/js/angular/clustermanagement/cluster-drawing.service.js]: Extracted in `NodeState` and `LinkState` objects

File: src/js/angular/clustermanagement/cluster-drawing.service.js (hunk 1 above)
Comment (yordanalexandrov) [reply src/js/angular/clustermanagement/cluster-drawing.service.js:177]: This is adding "padding" to the text background. I added a comment.

**Review by svilenvelikov — APPROVED** [review]

**Pull request merged**